examples/views/button_roles.py

```diff
@@ -57,17 +55,19 @@ async def callback(self, interaction: discord.Interaction):
 
 
 class ButtonRoleCog(commands.Cog):
-    """A cog with a slash command for posting the message with buttons
+    """
+    A cog with a slash command for posting the message with buttons
     and to initialize the view again when the bot is restarted
     """
 
     def __init__(self, bot):
         self.bot = bot
 
     # make sure to set the guild ID here to whatever server you want the buttons in
-    @slash_command(guild_ids=[846383887973482516], description="Post the button role message")
+    @slash_command(guild_ids=[...], description="Post the button role message")
     async def post(self, ctx: commands.Context):
-        """Slash command to post a new view with a button for each role
+        """
+        Slash command to post a new view with a button for each role
```

Comment:
```suggestion
        """Slash command to post a new view with a button for each role
```
Consistency with project code style.